docs/tutorials/gluon/hybrid.md

```diff
@@ -105,23 +105,35 @@ Hybridize will speed up execution and save memory. If the top level layer is
 not a `HybridBlock`, you can still call `.hybridize()` on it and Gluon will try
 to hybridize its children layers instead.
 
+`hybridize` also accepts several options for performance tuning. For example, you
+can do
+
+```python
+net.hybridize(static_alloc=True)
+# or
+net.hybridize(static_alloc=True, static_shape=True)
```

Comment:
This is slower in 1.2 https://github.com/apache/incubator-mxnet/pull/11293
let's not advertise it. I don't even think these flags exist in 1.2.0
let's just mention `net.hybridize()` in 1.2

Reply:
Removed.